scripts/script_requirements.txt

```diff
@@ -0,0 +1,2 @@
+click
+PyGithub
```

Comment:
Yes. I've been using `github3.py` in `repo-tools` because it seemed the most comprehensive and well documented and maintained.